core/store/orm/orm.go

```diff
@@ -899,6 +899,18 @@ func (orm *ORM) TxAttempts(offset, limit int) ([]models.TxAttempt, int, error) {
 	return attempts, count, err
 }
 
+func (orm *ORM) UnconfirmedTxAttempts() ([]models.TxAttempt, error) {
+	var items []models.TxAttempt
+	err := orm.DB.
+		Where("confirmed = false").
```

Comment:
@j16r @se3000 
Technically, this `confirmed` bool flag is only set when Safe, not on the first confirmation via a receipt. We've opted to use this anyways and rebroadcast txs that might be confirmed but not safe, but warn in that scenario, since that scenario could happen while we're offline anyways. ie:

1. tx A is sent, but unconfirmed
2. We go offline, but it gets confirmed
3. We reconnect and rebroadcast.

Ideally, we would want to rename this column `safe` and have a separate column `confirmed` to also identify the mere act of confirmation, or the reception of a tx receipt. That's been captured here: https://www.pivotaltracker.com/story/show/168536671